lib/dal-test/src/helpers.rs

```diff
@@ -420,11 +419,9 @@ pub async fn fetch_resource_last_synced_value(
         return Err(eyre!("unexpected: more than one attribute value found"));
     }
 
-    let last_synced_value = AttributeValue::get_by_id(ctx, attribute_value_id)
-        .await?
-        .view(ctx)
-        .await?;
-    Ok(last_synced_value)
+    AttributeValue::view(ctx, attribute_value_id)
+        .await
+        .map_err(Into::into)
```

Comment:
Is the `.map_err(Into::into)` here to avoid `Ok(<expr>?)`? The latter can end up looking better (and being fewer lines) if it works.